contracts/HypERC20Collateral.sol

```diff
@@ -36,6 +36,10 @@ contract HypERC20Collateral is TokenRouter {
         );
     }
 
+    function balanceOf(address _account) external view returns (uint256) {
+        return wrappedToken.balanceOf(_account);
+    }
+
```

Comment:
This is nice, before I had the UI needed extra logic to know to check the wrapped token instead of the collateral.